ptf/tests/common/fabric_test.py

```diff
@@ -2149,8 +2148,12 @@ def runIPv4MulticastTest(self, pkt, in_port, out_ports, in_vlan, out_vlan):
         # Send packets and verify
         self.send_packet(in_port, pkt)
         for out_port in out_ports:
+            if out_port == in_port:
+                # Packet will be dropped by egress pipeline.
+                continue
```

Comment:
Does that mean before we were expecting to receive a packet on `in_port`? If yes, since the pipeline performs inport pruning, how is it possible that the multicast test was passing before this change?

Reply:
For IPv4 multicast test, we didn't include ingress port to the egress port list, which means the clone group won't clone the packet to the ingress port
https://github.com/stratum/fabric-tna/blob/main/ptf/tests/unary/test.py#L118-L119